x-pack/filebeat/module/o365/audit/test/15-azuread-sts-logon.log-expected.json

```diff
@@ -0,0 +1,6350 @@
+[
+    {
+        "@timestamp": "2020-02-10T15:13:13.000Z",
+        "client.address": "83.57.233.151",
+        "client.ip": "83.57.233.151",
+        "event.action": "UserLoggedIn",
+        "event.category": "authentication",
+        "event.code": "AzureActiveDirectoryStsLogon",
+        "event.dataset": "o365.audit",
+        "event.id": "ca0efc24-1b89-4962-8fef-a3ac5437302f",
+        "event.kind": "event",
+        "event.module": "o365",
+        "event.outcome": "success",
+        "event.provider": "AzureActiveDirectory",
+        "event.type": [
+            "start",
+            "authentication_success"
```

Comment:
Question.  I don't think "authentication_success" is an official ECS event.type. I know it has been used in the past, but do we want to keep using it?

Reply:
I just added it because I thought the SIEM app would still be looking for it. In the future we can remove it from here and other places where it's used (auditbeat's system/login).

Reply:
makes sense.  thanks.